constants/Dictionary.ts

```diff
@@ -2,124 +2,125 @@ import * as Localization from 'expo-localization';
-        yourRequestHasNotYetBeenApproved:                   'your request has not yet been approved',
-        yourRequestHasAlreadyBeenApproved:                  'your request has already been approved',
-        username:                                           'username',
-        address:                                            'address',
-        reply:                                              'reply',
-        approved:                                           'approved',
-        pickADate:                                          'pick a date',
-        confirm:                                            'confirm',
-        unsupportedStatus:                                  'unsupported status',
-        unsupportedContent:                                 'unsupported content',
-        yourGoodsAreNotFound:                               'your goods are not found',
-        noGoodsAreAvailable:                                'no goods are available',
-        resultSetIsEmpty:                                   'result set is empty',
-        pleaseEnterAnItem:                                  'please enter an item',
-        expiryMustNotBeAPastDate:                           'expiry must not be a past date'
+    add:                                                    'add',
+    perishableGoods:                                        'perishable goods',
+    setLocation:                                            'set location',
+    bestBefore:                                             'best before',
+    successfullyAdded:                                      'successfully added',
+    letsContinueWithOtherPerishableGood:                    "let's continue with other perishable good",
+    cancel:                                                 'cancel',
+    okay:                                                   'Okay',
+    new:                                                    'New',
+    chooseAPhoto:                                           'choose a photo',
+    expirationDate:                                         'expiration date',
+    pickedLocation:                                         'picked location',
+    all:                                                    'Mine',
+    signUp:                                                 'sign up',
+    name:                                                   'name',
+    emailAddress:                                           'email address',
+    password:                                               'password',
+    confirmPassword:                                        'confirm password',
+    submit:                                                 'submit',
+    youHaveSuccessfullysignedUp:                            'you have successfully signed up',
+    aProblemOccurredWhileCommunicatingWithTheServer:        'a problem occurred while communicating wit h the server',
```

Comment:
This space breaks three tests, we have to remove it!

Reply:
The space between the ':' and the string value?

Reply:
No, I am sorry if I was not clear. The space is in the message:
> 'a problem occurred while communicating **wit h** the server'

Reply:
Ohh okay!

Reply:
The failed tests have gone away, now only the Sonar Cloud fails, just like in the first commit PR.